compiler/src/dotty/tools/dotc/transform/init/Objects.scala

```diff
@@ -489,8 +489,8 @@ object Objects:
     def read(addr: Addr)(using mutable: MutableData): Value =
       mutable.heap(addr)
 
-    def write(addr: Addr, value: Value)(using mutable: MutableData): Unit =
-      mutable.update(addr, value)
+    def writeUnion(addr: Addr, value: Value)(using mutable: MutableData): Unit =
```

Comment:
Without braces, it's hard to see which methods are part of `class MutableData` and which come after. Suggest an `end` line to indicate the end of `MutableData`.